### ISSUE TITLE
Call for new maintainers 

### ISSUE THREAD

**Issue by andyperlitch** [nikhilmodak/gulp-ngdocs#120]
Hello community,

This project is in need of some TLC by some new maintainers. I no longer have need for this project in or out of my day job and so my focus has been elsewhere. It appears to be the same for @nikhilmodak. There seems to still be a good amount of interest in this project and so I am hoping that someone who is vested in its success will take up the torch and continue improvement on it. Please respond if you are interested.

@kramar, @nikochaffin, @sp90, @brentoneill, @Billbastos, @chrats, @MaziMuhlari, @yaacovCR, @caseyhoward, @michaelbazos, @whitneyit, @manuelro

**Comment by caseyhoward** [comment]
I would be more interested in an exit strategy to use something that doesn't require gulp. This projects works good enough for me for the time being. However, I have no interest in maintaining it (especially as a gulp plugin). I would have to write a bunch of high level tests before I even considered it. I don't want to break things. That seems like a good amount of work that I don't have time for. I don't really use gulp anymore, but in the project I have that uses this project, it is the main reason I have to install gulp.

Does anyone have any ideas for another project for generating browsable docs for angular? It might not even have to be specific to angular. I haven't had any luck finding one I like.

**Comment by andyperlitch** [comment]
@caseyhoward [the dgeni project](https://github.com/angular/dgeni) was supposed to be the de facto angular doc generator. In the past I've tried using it and even considered refactoring this project to use it, but found that it lacked a lot out-of-the-box.

**Comment by caseyhoward** [comment]
Yep. I found the same problem too. It's pretty disappointing and frustrating. I wish the Angular team would release a full solution. They should release whatever they use. Nobody wants to dig through the source code of a documentation library to figure out how to use it then go manually write their own documentation generator and have to write their own HTML and CSS. I would like something that looks good up front, but I can customize later. It seems like a complete oversight to me.

**Comment by andyperlitch** [comment]
Couldn't agree more
